xcflushd.gemspec

```diff
@@ -24,19 +24,19 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.required_ruby_version = '>= 2.1.0'
+  spec.required_ruby_version = '>= 2.3.0'
 
-  spec.add_runtime_dependency "3scale_client", "~> 2.10"
-  spec.add_runtime_dependency "gli", "= 2.14.0"
-  spec.add_runtime_dependency "redis", "= 3.3.2"
-  spec.add_runtime_dependency "hiredis", "= 0.6.1"
-  spec.add_runtime_dependency "concurrent-ruby", "1.0.2"
-  spec.add_runtime_dependency "net-http-persistent", "2.9.4"
-  spec.add_runtime_dependency "daemons", "= 1.2.4"
+  spec.add_runtime_dependency "3scale_client", "~> 2"
```

Comment:
'~> 2.10.0' would lock to 2.10.X but I don't think we want that.
It changed this back to how it was.